code/datums/wires/wires.dm

```diff
@@ -121,15 +121,23 @@ var/list/wireColours = list("red", "blue", "green", "black", "orange", "brown",
 			var/obj/item/I = L.get_active_hand()
 			holder.add_hiddenprint(L)
 			if(href_list["cut"]) // Toggles the cut/mend status
-				if(iswirecutter(I))
+				var/switchtool_deployed
```

Comment:
Why not just set I to S.deployed and feed that to the code that was already there